### ISSUE TITLE
Cannot rank according to attribute data

### ISSUE THREAD

**Issue by mhockenhull** [gephi/gephi#1392]
Experiencing an issue with Gephi 0.9.1 where data from integer, double and float coloumns do not appear in the drop-down menu for ranking the nodes. I have also tried duplicating the data into a new coloumn to make sure that the data format is correct, but the issue persists. 

This occurs particularly with a network graph generated by the DMI-TCAT tool. I suspect that it may be something with the network file, as I get an error message stating that the GEXF 1.2 is deprecated. Could this be the issue?

**Comment by kooelo** [comment]
Can confirm this on a linux machine running Gephi 0.9. I wanted to use a numeric attribute for node coloring, but this variable is not visible for ranking.

**Comment by matjemeisje** [comment]
I also have this problem on Mac with Gephi 0.9.1.

**Comment by eduramiba** [comment]
I think this is a duplicate of https://github.com/gephi/gephi/issues/1459

**Comment by matjemeisje** [comment]
No, it is not. The solution proposed there does not solve my issue. Have just tried it in the 0.9.2 nightly build with the same results.

**Comment by eduramiba** [comment]
Does it contain any null value? That might be a problem.

**Comment by matjemeisje** [comment]
No, no null values. It might be of interest that the graph was created using the Twitter Streaming Importer plugin. But this problem does not occur with every graph imported with the plugin. I have given an assignment to a group of students and for most of them it works, but some don't. I have experienced it myself on Mac on and off. The current graph was made by a student on Windows, I believe, and sent to me. But both OS display the problem.
I just tested it also in 0.8.1 and there it does work. I notice one difference in the Data Laboratory between versions: 0.8.1 displays the centrality measures without trailing zeros (1), and 0.9 with trailing zeros (1.0). Maybe it's something to do with the data type used internally by Gephi?
